verify/v1/verify.yaml

```diff
@@ -393,55 +422,60 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/error500'
-  /phone/{number}?criteria={criteriaName}:
+  /phone/{number}?criteria={criteriaTemplateName}:
     get:
       tags:
-        - "Verify Criteria"
-      summary: Verify criteria by name
+        - "Verify with Criteria Templates"
+      summary: Verify by criteria template
       description: |
-        Verify criteria by criteria name. Customers can view a specific response by setting up response parameter at the time of criteria creation.
+        Verify a phone number based on a criteria.
+        
+        Each criteria has a defined verify and response parameters, set at the criteria creation api request. 
+        Verify request is based on the defined criteria for providing feedback regarding the requested phone number. 
+        
+        Example of a successful Verify response is shown at the code section at the right side of this page.
+        
+        Unverified requests do not respect the defined parameter set in the response of the criteria, but they are using
+        a default set of response parameters, aiming to explain to the customer the reason that the request was unverified.
+        The unverified request response will always show the following four parameters:
         
-        Unverified responses will always show the following four parameters:
         - requestId
         - verified
         - reason
         - reasonCode
         
         Depending on the reason that the request is unverified, the following error codes and reasons will be sent back:
```

Comment:
should we say "reason codes" instead of "error codes" to be consistent with the below table ?